BoxContentSDK/BoxContentSDK/Requests/BOXFolderPaginatedItemsRequest.m

```diff
@@ -90,12 +116,21 @@ - (void)performRequestWithCompletion:(BOXItemArrayCompletionBlock)completionBloc
                                                                                       ancestors:pathFolders];
             }
             [BOXDispatchHelper callCompletionBlock:^{
-                completionBlock(items, totalCount, NSMakeRange(offset, limit), nil);
+                refreshBlock(items, totalCount, NSMakeRange(offset, limit), nil);
+                
+                // Don't want to update cache for a subclass recursively calling this method.
```

Comment:
I think it would be good to update the cache in this case. It's a valid cache entry as long as the cache key contains the offset and limit somehow.

We just need to make sure these cache entries have zero impact on the caching of the subclass (BOXFolderItemsRequest) which should be doing its own caching.